### DISCUSSION TITLE
ChakraUI v2 ChakraProvider causes ChunkLoadError in app/layout

### DISCUSSION THREAD

**Discussion by dberruezo** [chakra-ui/chakra-ui#9213] [General] [answered]
### Description

When using the latest version of ChakraUI v2 with Next v14@latest and just add the ChakraProvider as the docs indicate then run the dev server you are presented with a ChunkLoadError in app/layout. Upon inspection the layout.js is received truncated by the browser. Not to mention this also breaks the entire hydration.

### Link to Reproduction

https://github.com/dberruezo/chakraui-v2-layout-error

### Steps to reproduce

1. Install Next v14 with Node v20
2. Install using the official instructions https://v2.chakra-ui.com/getting-started/nextjs-app-guide up to the provider.
3. npm run dev
4. Open console and enter localhost:3000
5. The console shows the error

### Chakra UI Version

2.10.4

### Browser

Chrome 129

### Operating System

- [ ] macOS
- [ ] Windows
- [X] Linux

**Comment by segunadebayo** [comment, marked as answer]
v2 is in maintenance. We are not shipping new v2 work. Stay on 2.8.x or migrate to v3.
